### ISSUE TITLE
"Visible watermark" does not work with CJK text

### ISSUE THREAD

**Issue by hexchain** [dtschump/gmic#282]
Using CJK text as watermark results in something that looks like an encoding error:

![image](https://user-images.githubusercontent.com/529520/104121514-1f69bb00-533f-11eb-8ae4-a72a5920ddf9.png)

This is G'MIC 2.9.4 and GIMP 2.10.22.

**Comment by dtschump** [comment]
Yes, unfortunately, the character set supported by the G'MIC command `text` is quite limited at the moment (and does absolutely not support UTF-8). Command `fontchart` shows the available characters.
Every filter that is based on command `text` will have this limitation.

![fontchart](https://user-images.githubusercontent.com/10114565/110105735-0ca4bd00-7da9-11eb-9508-5deacbf8905f.jpg)
